charts/kserve-resources/values.yaml

```diff
@@ -76,6 +76,20 @@ kserve:
 
     # -- KServe controller manager rbac proxy contrainer image
     rbacProxyImage: quay.io/brancz/kube-rbac-proxy:v0.18.0
+    rbacProxy:
+      resources:
+        limits:
+          cpu: 100m
+          memory: 300Mi
+        requests:
+          cpu: 100m
+          memory: 300Mi
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+          - ALL
+        readOnlyRootFilesystem: true
```

Comment:
I am not sure if you also need to add `runAsNonRoot: true` into the securityContext.

Reply:
added 👍